### PR TITLE
Modify overlays

### PR DESCRIPTION
**Pull request by piensaenpixel**

### CODE REVIEW

**Comment by piensaenpixel** [comment]
@alonsogarciapablo could you take a look?

image



cc @xavijam

File: examples/v4/leaflet-layergroup.html [hunk 1]
```diff
@@ -46,7 +46,7 @@
           "zoom":2,
           "updated_at":"2015-03-13T11:24:37+00:00",
           "datasource": {
-            "user_name": "documentation",
+            "user_name": "granemil",
```
Comment (alonsogarciapablo) [inline examples/v4/leaflet-layergroup.html]: :cachondo:

File: src/geo/map-view.js [hunk 2]
```diff
@@ -1,6 +1,8 @@
 var _ = require('underscore');
 var log = require('cdb.log');
 var View = require('../core/view');
+var overlayTemplate = require('./ui/overlay.tpl');
```
Comment (alonsogarciapablo) [inline src/geo/map-view.js]: `var overlaysContainerTemplate = require('./ui/overlays-container.tpl');` ?

File: src/geo/map-view.js [hunk 3]
```diff
@@ -52,8 +55,14 @@ var MapView = View.extend({
   },
 
   addOverlay: function (overlay) {
+    var type;
     if (overlay) {
-      this.$el.append(overlay.render().el);
+      type = overlay.type;
+      if (type && CONTAINED_OVERLAYS.indexOf(type) >= 0) {
+        this._overlayContainer().append(overlay.render().el);
+      } else {
```
Comment (alonsogarciapablo) [inline src/geo/map-view.js]: What other types of overlays do we have? Why not add them all to the same container? Individual overlays can be positioned absolutely via CSS anyway...

File: examples/v4/leaflet-layergroup.html (hunk 1 above)
Comment (piensaenpixel) [inline examples/v4/leaflet-layergroup.html]: HAHAHAHA sorry

File: src/geo/map-view.js (hunk 3 above)
Comment (piensaenpixel) [inline src/geo/map-view.js]: for example loader cc/@nobuti

File: themes/scss/map/attributions.scss [hunk 4]
```diff
@@ -5,60 +5,51 @@ $size: 20px;
...
+  vertical-align: middle;
 }
 .CDB-Attribution-text {
   @include transition(opacity, 150ms);
-  pointer-events: none;
-  color: #FFF;
-  font-size: $sFontSize-medium;
-  line-height: $size;
-  opacity: 0;
+  border-left: 1px solid transparent;
```
Comment (xavijam) [inline themes/scss/map/attributions.scss]: hound is right

File: themes/scss/map/attributions.scss [hunk 5]
```diff
@@ -5,60 +5,51 @@ $size: 20px;
...
-  width: $size;
-  height: $size;
-  border: 1px solid rgba(0, 0, 0, 0.2);
-  border-radius: 50%;
-  background: #FFF;
-  color: #666;
-  font-size: $sFontSize-medium;
-  line-height: $size;
-  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.3);
+  color: #636D72;
```
Comment (xavijam) [inline themes/scss/map/attributions.scss]: hound is right

**Comment by xavijam** [comment]
I encourage you (both) to review style problems, please.

**Comment by piensaenpixel** [comment]
@xavijam @alonsogarciapablo  take a look :) i fixed overlays for mobile view

overlays

**Comment by xavijam** [comment]
:es:

**Comment by alonsogarciapablo** [comment]
👍

**Pull request merged**